controllers/metrics.go

```diff
@@ -34,7 +34,7 @@ var (
 			Namespace: "projectsveltos",
 			Name:      "program_resources_time_seconds",
 			Help:      "Program Resources on a workload cluster duration distribution",
-			Buckets:   []float64{1, 10, 30, 60, 120, 180, 240},
+			Buckets:   []float64{0.5, 0.55, 0.6, 0.65, 0.7, 1, 1.5, 2, 5, 10, 60},
```

Comment:
Thank you. While this is are correct with our data, just in case users deployed lots of resources what do you think of relying just to seconds, so something something like 

{0.5, 1, 1.5, 2, 3, 5, 10, 20, 40, 60, 120} ?

Just in case someone has a management cluster in Europe and managed in US

Reply:
Just changed that!